src/workflows-app/SubmissionDetails.js

```diff
@@ -234,9 +243,34 @@ export const BaseSubmissionDetails = ({ name, namespace, workspace, submissionId
                   ),
               ]),
             ]),
+            h(Tabs, {
```

Comment:
Can we try to align the tabs component with the rest of the elements page? I noticed it's skewed to the left and was hoping to align it with the details in the header.